Rakefile

```diff
@@ -1,6 +0,0 @@
-# Add your own tasks in files placed in lib/tasks ending in .rake,
```

Comment:
why delete this file?

Reply:
I didn't delete. Must be the result of some weird merge conflict. I've been seeing those a lot lately.

Reply:
Yeah, I even looked on my computer. That line is there. It's 100% not deleted.